libs/designsystem/src/lib/components/button/button.component.spec.ts

```diff
@@ -19,6 +19,24 @@ describe('ButtonComponent', () => {
     declarations: [MockComponent(IconComponent)],
   });
 
+  const canvasColor = 'background-color';
+
+  beforeAll(() => {
```

Comment:
Would a comment explaining why this is necessary be appropriate here?

Reply:
The test renders a button only with no context. That means, that the `color` property of the button will inherit its value from the document. The initial value (defined by the spec) for `color` is `#000000` - black, not kirby-black.

When kirby-primary is light e.g., `background-color` then kirby-primary-contrast will be kirby-black. This makes the assertions below (lines ~131 and ~245) fail because kirby-black is not completely black (#000000).

This could be left-overs from an earlier iteration and may be obsolete. I'll look into it 😊

Reply:
> The test renders a button only with no context. That means, that the `color` property of the button will inherit its value from the document. The initial value for `color` is black (`#000000`), not kirby-black (`#1c1c1c`). (The initial value for color varies from one browser to another and in our case it's black).
> 
> When kirby-primary is light e.g., `background-color` then kirby-primary-contrast will be kirby-black. This makes the assertions below (lines ~131 and ~245) fail because kirby-black is not completely black (`#000000`).
> 
> This could be left-overs from an earlier iteration and may be obsolete. I'll look into it 😊

It is no longer necessary to arrange a canvas/context. Attention levels 3-4 have `background-color: transparent` and assume to always be displayed on a light background. Hence the `color` is always kirby-black regardless of brand colors (see https://github.com/kirbydesign/designsystem/blob/8c62cfef29a76c578cb10b634232d2b2c38f2c78/libs/designsystem/src/lib/components/button/button.component.scss)

I have removed it.